makim/__main__.py

```diff
@@ -180,6 +176,9 @@ def app():
     args_parser = _get_args()
     args = args_parser.parse_args()
 
+    if not args.target and not args.help:
```

Comment:
```suggestion
    if not args.target or args.help:
```

probably you could have just one "if" for the help.

Reply:
I don't know if I'm following you, they are different cases, a `--help` can be used with a target, can't it?

Reply:
Nope, the help will show the info for all groups and targets in 1 shot

Reply:
but how it would be? `(if not target and help) or (not target and not help)`, something like this?

Reply:
I think that `not target or help` would do the trick